lib/commands/gesture.js

```diff
@@ -0,0 +1,43 @@
+import logger from '../logger';
+
+let commands = {};
+
+/**
+ * Perform touch command
+ */
+commands.performTouch = async function (gestures) {
+  //Convert and pass on the touch command
+  if (gestures.length === 1 && gestures[0].action === 'tap') {
+    return await this.handleTap(gestures[0]);
+  }
+  //TODO: Handle multiple gestures, or anything else but tap
```

Comment:
Instead of this TODO, can you throw a NotImplemented Error with a helpful comment?